tools/cpanfile.additions

```diff
@@ -0,0 +1,5 @@
+# Additional hardcoded lines for cpanfile
```

Comment:
can you say what makes these lines "hardcoded"? Why does the yaml file not work for specifying dependencies?